test/node/long.test.ts

```diff
@@ -163,4 +163,101 @@ describe('Long', function () {
       });
     });
   });
+
+  describe('static fromStringStrict()', function () {
+    const successInputs: [
+      name: string,
+      input: string,
+      unsigned: boolean,
+      radix: number,
+      expectedStr?: string
+    ][] = [
+      ['basic no alphabet low radix', '1236', true, 8],
+      ['negative basic no alphabet low radix', '-1236', false, 8],
+      ['valid upper and lower case letters in string with radix > 10', 'eEe', true, 15],
+      ['hexadecimal letters', '126073efbcdADEF', true, 16],
+      ['negative hexadecimal letters', '-1267efbcdDEF', false, 16],
+      ['negative leading zeros', '-00000032', false, 15, '-32'],
+      ['leading zeros', '00000032', false, 15, '32'],
+      ['explicit positive leading zeros', '+00000032', false, 15, '32'],
```

Comment:
explicit positive without any leading zeros

Reply:
Yep, added it!